pkgs/applications/networking/cluster/helm/plugins/default.nix

```diff
@@ -14,4 +13,5 @@
 
   helm-unittest = callPackage ./helm-unittest.nix { };
 
+  helm-mapkubeapis = callPackage ./helm-mapkubeapis.nix { };
```

Comment:
Please keep the file alphabetically sorted

Reply:
It's not already sorted though hence why I added it to the bottom. Let me sort the file itself.